app/src/main/kotlin/com/github/funkyg/funkytunes/service/TorrentManager.kt

```diff
@@ -103,6 +106,29 @@ class TorrentManager(private val context: Context) : AlertListener {
 						preloadNextTrack(position, alert.handle())
 					}
 				}
+				AlertType.BLOCK_FINISHED -> {
```

Comment:
its probably better if you use PIECE_FINISHED instead, because that is called less often (better for performance/battery usage)